OuterWildsRandomSpeedrun/OuterWildsRandomSpeedrun.cs

```diff
@@ -62,26 +68,44 @@ private void Start()
 
         private void Update()
         {
-            if (_isGameStarted && _shouldWarp)
-            {
+            if (!_isGameStarted || !_modEnabled) {
+                return;
+            }
+
+            if (_shouldStartTimer) {
+                _shouldStartTimer = false;
+                _startTimeMillis = DateTime.Now.Ticks / TimeSpan.TicksPerMillisecond;
+            }
+
+            if (_shouldWarp) {
                 HandleBasicWarp();
-                ShowTimerText();
             }
+            
+            var elapsed = TimeSpan.FromMilliseconds(DateTime.Now.Ticks / TimeSpan.TicksPerMillisecond - _startTimeMillis);
```

Comment:
Looks like you can get a TimeSpan by subtracting two DateTime objects. Thoughts on storing those instead of a `long` for additional type safety?